src/spring/setup.py

```diff
@@ -16,7 +16,7 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '1.11.3'
+VERSION = '1.13.0'
```

Comment:
Can you explain why you need to skip version `1.12.0`?

Reply:
OK, use version 1.12.0 here.